pkgs/top-level/aliases.nix

```diff
@@ -892,6 +892,7 @@ mapAliases ({
   stanchion = throw "Stanchion was part of riak-cs which is not maintained anymore"; # added 2020-10-14
   stumpwm-git = throw "stumpwm-git has been broken for a long time and lispPackages.stumpwm follows Quicklisp that is close to git version"; # added 2021-05-09
   surf-webkit2 = surf; # added 2017-04-02
+  svgcleaner = throw "svgcleaner has been removed."; # added 2021-11-17
```

Comment:
The message should contain a reason like it is abandoned or broken or similar.

Reply:
@SuperSandro2000 Should I push the same branch here again to add that or open a new pull request?

Reply:
Open a new pull request will be nice

Reply:
You can push to the same branch but you would need to open a new PR regardless of the branch name.

Reply:
@jgarte should have added reason. btw, svgcleaner still works, so i think removing it from nixpkgs was unnecessary.